llama_stack/distribution/routers/routing_tables.py

```diff
@@ -222,11 +223,21 @@ async def register_model(
                 )
         if metadata is None:
             metadata = {}
+        if model_type is None:
+            model_type = ModelType.llm
+        if (
+            "embedding_dimension" not in metadata
```

Comment:
oh you already have this here, then you should not need to specify it in memory banks

Reply:
Yes, the only reason for having that there is for convinience of not having to keep querying the registry for every call for embedding to get the metadata for the model.